extension/storage/filestorage/client_test.go

```diff
@@ -21,13 +20,13 @@ import (
 func TestClientOperations(t *testing.T) {
 	dbFile := filepath.Join(t.TempDir(), "my_db")
 
-	client, err := newClient(zap.NewNop(), dbFile, time.Second, &CompactionConfig{}, false)
+	client, err := newClient(zap.NewNop(), dbFile, time.Second, 0, &CompactionConfig{}, false)
 	require.NoError(t, err)
 	t.Cleanup(func() {
-		require.NoError(t, client.Close(context.TODO()))
+		require.NoError(t, client.Close(t.Context()))
```

Comment:
I made many updates to the tests in this package to resolve linter/modernize findings

Reply:
You can submit these in a separate PR and we can merge them independently, if you want.